### PR TITLE
merge dev main

### PR DESCRIPTION
**Pull request by ozen2**

### CODE REVIEW

**Review by lxndrMr — APPROVED** [review]

File: client/src/assets/images/carlos-kenobi-Knla5kYvsRE-unsplash.jpg
Comment (FabienRollet) [inline client/src/assets/images/carlos-kenobi-Knla5kYvsRE-unsplash.jpg:1]: No use for this pic anymore

File: client/src/assets/images/daniel-olah-16XJMQ2bTl4-unsplash.jpg
Comment (FabienRollet) [inline client/src/assets/images/daniel-olah-16XJMQ2bTl4-unsplash.jpg:1]: same

File: client/src/assets/images/pawel-czerwinski-fAd4u_HTGGY-unsplash (1).jpg
Comment (FabienRollet) [inline client/src/assets/images/pawel-czerwinski-fAd4u_HTGGY-unsplash (1).jpg:1]: same

File: client/src/assets/images/solen-feyissa-AI9Z8vNLcKc-unsplash.jpg
Comment (FabienRollet) [inline client/src/assets/images/solen-feyissa-AI9Z8vNLcKc-unsplash.jpg:1]: same (for this file and the next one

File: client/src/components/TopPlanet.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import PropTypes from "prop-types";
+import Links from "./Links";
+import Slogan from "./Slogan";
+import "../styles/Header.scss";
+
+function TopPlanet({ introPlanet }) {
+  const [thumbnailUrl, setThumbnailUrl] = useState(" ");
+  const [titleUrl, setTitleUrl] = useState(" ");
+
+  fetch("https://api.star-citizen.wiki/api/galactapedia/VarZxj1Qav")
```
Comment (pjroggy) [inline client/src/components/TopPlanet.jsx]: change id of planet to Rvl7EggdAr

Comment (ozen2) [inline client/src/components/TopPlanet.jsx]: change this id

**Review by pjroggy — CHANGES_REQUESTED** [review]

**Review by FabienRollet — APPROVED** [review]

**Review by pjroggy — APPROVED** [review]

**Review by lxndrMr — APPROVED** [review]

File: client/src/components/JourneyDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import { ShipContext } from "../Contexts/ShipContext";
+import "../styles/JourneyDetails.scss";
+
+export default function JourneyDetails() {
+  const { reservationFormData } = useContext(ReservationContext);
+  const { isButtonVisible, setIsButtonVisible } = useContext(ButtonContext);
+  const { ships } = useContext(ShipContext);
+  const { selectedShipIndex, selectedShipsData } = reservationFormData;
+
+  const handleClick = () => {
```
Comment (Yoann-Fortin) [inline client/src/components/JourneyDetails.jsx:21]: Dead code

File: client/src/pages/Booking.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import JourneyDetails from "../components/JourneyDetails";
+import ReservationModule from "../components/ReservationModule";
+import VehiculeSelect from "../components/VehiculeSelect";
+
+export default function Booking() {
+  return (
+    <main>
```
Comment (Yoann-Fortin) [inline client/src/pages/Booking.jsx:7]: Both main, the first is in App.jsx

File: client/src/pages/ErrorPage.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { Link } from "react-router-dom";
+import "../styles/ErrorPage404.scss";
+
+function ErrorPage404() {
+  return (
+    <body className="error404Body">
```
Comment (Yoann-Fortin) [inline client/src/pages/ErrorPage.jsx:6]: Not possible

**Pull request merged**